frontend/app/pages/about_us.vue

```diff
@@ -0,0 +1,116 @@
+      class="lg:h-[320px] md:h-[180px] relative flex flex-col justify-center items-center w-full overflow-hidden
+             sm:text-center sm:items-center sm:px-0 text-left"
+    >
+      <UContainer class="flex flex-col justify-center items-center">
+        <h1 class="heading-h3 font-semibold text-lg">
+          Hack4Krak
+        </h1>
+        <p class="max-w-[420px]">
+          to inicjatywa młodych liderów cyberbezpieczeństwa, których połączyła pasja do technologii, wyzwań i ciągłego
+          rozwoju. Tworzymy wydarzenia, które uczą, inspirują i integrują młodzież zainteresowaną światem IT i
+          bezpieczeństwa cyfrowego.
+        </p>
+      </UContainer>
+      <p
+        class="lg:text-[320px] md:text-[220px] text-[180px] font-semibold absolute -z-10 text-primary
+               stroked-text-full-screen w-screen md:w-auto text-wrap"
+      >
+        HACK4KRAK
+      </p>
+    </section>
+    <section
+      id="timeline"
+      class="relative flex justify-center items-center"
+    >
+      <div class="flex md:flex-row flex-wrap flex-col gap-8 items-center justify-center">
+        <EventCard v-for="card in aboutUsTimeline" v-bind="card" :key="card.title" />
+      </div>
+      <NuxtImg
+        class="absolute h-[300px] left-0 md:top-auto top-[200px] -z-20" src="/img/blockAccentVector.svg"
+        alt="block accent vector"
+        height="300px"
+      />
+      <NuxtImg
+        class="absolute h-[300px] right-0 md:top-auto top-[600px] -z-20 rotate-180" src="/img/blockAccentVector.svg"
+        alt="block accent vector"
+        height="300px"
+      />
+    </section>
+    <section id="mission">
+      <UContainer class="flex lg:flex-row flex-col-reverse mx-auto gap-12">
+        <div class="flex-1 flex flex-col gap-4 sm:px-0">
+          <h3 class="heading-h3 text-lg font-semibold">
+            Nasza misja
+          </h3>
+          <p>
+            Naszą misją jest inspirowanie i wspieranie młodych ludzi w rozwijaniu pasji do cyberbezpieczeństwa. Tworzymy
+            przestrzeń opartą na:
+          </p>
+          <div class="flex flex-wrap gap-4">
+            <Label>zaangażowaniu</Label>
+            <Label>praktycznej nauce</Label>
+            <Label>dzieleniu się wiedzą</Label>
+            <Label>rozwoju kompetencji przyszłości</Label>
+          </div>
+          <p>
+            Wierzymy, że edukacja w zakresie bezpieczeństwa cyfrowego powinna być dostępna, motywująca i prowadzona
+            przez
+            tych, którzy najlepiej rozumieją swoje pokolenie.
+          </p>
+        </div>
+        <div class="flex-1 justify-center items-center hidden sm:flex">
+          <NuxtImg class="w-full px-6 sm:px-0" src="/img/flag_n_clouds.webp" />
+        </div>
+      </UContainer>
+    </section>
+    <section id="mini-gallery">
+      <UContainer class="grid lg:grid-cols-4 lg:grid-rows-[auto, auto] grid-cols-3 gap-4">
+        <NuxtImg class="gallery-small-image" src="/img/about-us/Gallery3.webp" />
```

Comment:
wtf is that, why we cannot create a list of images and use it here instead of copying and creating random classes